server/src/serializers/PodcastSerializer.js

```diff
@@ -1,6 +1,6 @@
 class PodcastSerializer {
   static getSummary(podcast) {
-    const allowedAttributes = ["id", "name", "description"]
+    const allowedAttributes = ["id", "name", "description", "reviews"]
```

Comment:
We don't need to add reviews here in the allowed attributes, instead we can permit the properties we want for a podcast, serialize that record, then run our related query to find the associated reviews - this would be a great time to also create a ReviewSerializer! remove those timestamps unless they're being used on the webpage